scripts/check-events

```diff
@@ -34,12 +34,13 @@ end
 
```

Comment:
Unrelated to your code, this script is getting complex enough to warrant its own test suite.

Reply:
Want me to open an issue to track that ?

Reply:
I'd 👍🏻 that

Reply:
Opened: #14182